docs/README.md

```diff
@@ -0,0 +1,107 @@
+# Octokit Requirements
+
+> Provide a first class experience for platform developers in their preferred programming languages
+
+Octokit strives to offer complete and officially supported SDKs, quick starts, code samples, and copy/paste-able examples for the GitHub Platform in the most popular languages.
+
+Each Octokit will work towards meeting these requirements:
+
+|                                 | JavaScript | Ruby | .NET |
+| ------------------------------- | ---------- | ---- | ---- |
+| [Complete](#complete)           | 🌔         | 🌔   | 🌔   |
+| → REST                          | ✅         | ✅   | 🌔   |
+| → GraphQL                       | 🌔         | 🌑   | 🌔   |
+| → GitHub Apps                   | 🌔         | 🌔   | ✅   |
+| → OAuth                         | 🌔         | ✅   | ✅   |
```

Comment:
@kytrinyx does octokit.rb include rack middleware for OAuth? Or what is the current preferred method for implementing OAuth in Ruby?